FiveCalls/FiveCallsUITests/FiveCallsUITests.swift

```diff
@@ -13,7 +13,7 @@ class FiveCallsUITests: XCTestCase {
 
     var app: XCUIApplication!
         
-    override func setUp() {
+    @MainActor override func setUp() {
```

Comment:
Was this required? Just curious what happened without it...

Reply:
This was required with the new fastlane snapshot helper, the rest of that update will be in the following PR